parm/config/gfs/config.resources.WCOSS2

```diff
@@ -5,6 +5,7 @@
 case ${step} in
   "prep")
     export is_exclusive=True
+    export tasks_per_node=5
```

Comment:
Same question as for `parm/config/gfs/config.resources`.  Can we run JEDI apps with multiple threads?  If _yes_ this is great news!

Reply:
This was a misplaced comment.  The job in question is `prep`, not a JEDI app.